src/test/resources/mockData/srs/get_records_response.json

```diff
@@ -1,15 +1,9 @@
 {
-  "records": [
+  "sourceRecords": [
     {
-      "id": "bc5caa84-e85c-4c98-9956-50ccdaad7115",
+      "recordId": "bc5caa84-e85c-4c98-9956-50ccdaad7115",
```

Comment:
It is irrelevant for mock data, but just so you know, SRS now have constraints that ensure recordId, rawRecordId and parsedRecordId are the same

Reply:
Done